blog/en/blog/2022/12/06/choose-the-right-api-style-technology.md

```diff
@@ -0,0 +1,140 @@
+---
+title: How to choose the right API Style and Technology
+authors:
+  - name: Bobur Umurzokov
+    title: Author
+    url: https://github.com/Boburmirzo
+    image_url: https://avatars.githubusercontent.com/u/14247607
+keywords:
+  - API
+  - Style
```

Comment:
😅 This keyword is too general IMO, can pick another related one?

Reply:
Okay, this updated list makes sense to you?

  - API Style
  - Rest
  - Event-driven APIs
  - API Technologies
  - API resources